packages/lib/jobs/definitions/internal/seal-document.ts

```diff
@@ -127,7 +136,11 @@ export const SEAL_DOCUMENT_JOB_DEFINITION = {
       flattenForm(pdfDoc);
       flattenAnnotations(pdfDoc);
 
-      if (certificateData) {
+      if (
```

Comment:
This probably could have been a ternary on `certificateData` e.g.

```typescript
const certificateData =
      document.team?.teamGlobalSettings?.includeSignerCertificate === false
        ? null
        : await getCertificatePdf({ documentId }).catch(() => null);
```

Reply:
Then you wouldn't need the below which seems like it would break on individual documents anyways 🤔